tox.ini

```diff
@@ -0,0 +1,80 @@
+[tox]
+minversion = 2.0
+envlist = py{27,35,36,37}-{test,integration,benchmark}, sdist, flake8
+
+[pytest]
+addopts = -rfxEXs --durations=20 --showlocals --reruns 3 --reruns-delay 1
+
+[flake8]
+ignore = E12, W503, E226
+max-line-length = 110
+show-source = True
+
+
+[testenv]
+recreate = True
+whitelist_externals =
+    sh
+    bash
+
+deps =
+    .[test]
+
+    integration: numpy
+
+    benchmark: pytest_benchmark
+    benchmark: awscli
+
+setenv =
+    SO_DISABLE_MOCKS={env:SO_DISABLE_MOCKS:}
+    SO_DISABLE_MOTO_SERVER={env:SO_DISABLE_MOTO_SERVER:}
```

Comment:
This variable is now SO_ENABLE_MOTO_SERVER: https://github.com/RaRe-Technologies/smart_open/blob/master/smart_open/tests/test_s3.py#L35

Also, is this environment variable being used anywhere?

Reply:
fixed 73476c5